pow-migration/src/main.rs

```diff
@@ -164,9 +173,56 @@ async fn main() {
                 std::process::exit(1);
             }
         };
-        println!("Registered validators:");
-        for validator in registered_validators {
-            println!("{}", validator.validator.validator_address);
+
+        if args.list_validators {
+            println!("Registered validators:");
+            for validator in registered_validators {
+                println!("{}", validator.validator.validator_address);
+            }
+        } else {
+            if pow_client.block_number().await.unwrap()
```

Comment:
You're using the "unwrap or process exit" pattern quite a lot. You could either use `.expect(<string>)` or it could be extracted into a function:
```rs
fn unwrap_or_exit<T, E>(message: &'static str, result: Result<T, E>) -> T {
    match result {
        Ok(t) => t,
        Err(error) => {
            log::error(?error, "{}", message);
            std::process::exit(1);
        }
    }
}
```

Reply:
Addressed in 763cd1f.